mne/viz/_3d.py

```diff
@@ -408,8 +408,23 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
     surfs = dict()
     if head:
         subjects_dir = get_subjects_dir(subjects_dir, raise_error=True)
-        surfs['head'] = get_head_surf(subject, source=source,
-                                      subjects_dir=subjects_dir)
+        try:
+            surfs['head'] = get_head_surf(subject, source=source,
+                                          subjects_dir=subjects_dir)
```

Comment:
I would introduce a private function

_has_head_surf(...)

that is called by get_head_surf so we can avoid the try block